src/configLoader/getContent.js

```diff
@@ -61,3 +61,25 @@ function getConfigContent (configPath, baseDirectory) {
     const content = readConfigContent(resolvedPath);
     return getNormalizedConfig(configBasename, content);
 };
+
+/**
+ * Read proper content from config file.
+ * If the chartset of the config file is not utf-8, one error will be thrown.
+ * @param {String} configPath 
+ * @return {String}
+ */
+function readConfigFileContent (configPath) {
+
+    if (!configPath) {
+        throw new Error("config file path couldn't be empty");
+    }
+
+    let rawBufContent = fs.readFileSync(configPath);
+    let tmpBuf = Buffer.from(rawBufContent.toString("UTF-8"), "UTF-8");
+
+    if (!rawBufContent.equals(tmpBuf)) {
+      throw new Error(["We expect the charset of the config file is UTF-8, but", configPath, "is not , please check"].join('\n'));
+    }
+
+    return rawBufContent.toString("UTF-8");
```

Comment:
The preferred encoding is `'utf8'` as specified in the Node.js manual, let's use that instead of `"UTF-8"`